crates/aws_http_client/Cargo.toml

```diff
@@ -22,7 +22,7 @@ http_client.workspace = true
 schemars = { workspace = true, optional = true }
 strum.workspace = true
 thiserror.workspace = true
-gpui_tokio.workspace = true
+tokio = { workspace = true, features = ["rt", "rt-multi-thread"] }
```

Comment:
Let's update the dependency both here and in `gpui_tokio` to put these features in the `Cargo.toml`, so we can keep an eye on feature unification. I should have done that in the original PRs :)